spec/asyncapi.md

```diff
@@ -1412,7 +1412,6 @@ my.org.User
   },
```

Comment:
You should also remove it from https://github.com/asyncapi/spec/blame/0581e523d9e6fba1826ca4f529ee1bef94928df2/spec/asyncapi.md#L1487